sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyClientTest.java

```diff
@@ -69,7 +75,22 @@ protected void createKeyClient(HttpClient httpClient, KeyServiceVersion serviceV
             when(asyncClient.getDefaultPollingInterval()).thenReturn(Duration.ofMillis(10));
         }
 
-        keyClient = new KeyClient(asyncClient);
+        keyClient = new KeyClient(new KeyClientImpl(asyncClient.getVaultUrl(), asyncClient.getHttpPipeline(),
+            serviceVersion));
+    }
+
+    private HttpClient buildSyncAssertingClient(HttpClient httpClient) {
```

Comment:
We shouldn't skip these tests. We want to make sure things are still working well with the current implementation. Can we get away with not using `AssertingHttpClientBuilder` until polling and paging are supported?